src/components/RichTextEditor.tsx

```diff
@@ -121,6 +122,10 @@ const Styles = styled("div", {
         }
       : {},
   },
+  ".tox.tox-tinymce.tox-fullscreen": {
+    zIndex: 1203,
+    backgroundColor: "hsl(240, 9%, 7%)",
```

Comment:
```suggestion
    "& .tox-tinymce.tox-fullscreen": {
      zIndex: theme.zIndex.modal,
      backgroundColor: theme.palette.background.paper,
    },
```
In the light theme, the background color is black and the text is black, making it unreadable. Please use MUI theme variables to ensure consistency throughout the app.

Once this change has been committed, this PR is good to merge.

Reply:
Ok